libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -142,8 +142,8 @@ const AP_Param::GroupInfo AP_ICEngine::var_info[] = {
 
     // @Param: OPTIONS
     // @DisplayName: ICE options
-    // @Description: Options for ICE control. The DisableIgnitionRCFailsafe option will cause the ignition to be set off on any R/C failsafe. If ThrottleWhileDisarmed is set then throttle control will be allowed while disarmed for planes when in MANUAL mode.
-    // @Bitmask: 0:DisableIgnitionRCFailsafe,1:DisableRedineGovernor,2:ThrottleWhileDisarmed
+    // @Description: Options for ICE control. The DisableIgnitionRCFailsafe option will cause the ignition to be set off on any R/C failsafe. If ThrottleWhileDisarmed is set then throttle control will be allowed while disarmed for planes when in MANUAL mode. If disable while disarmed is set the engine will not start while the vehicle is disarmed.
+    // @Bitmask: 0:DisableIgnitionRCFailsafe,1:DisableRedineGovernor,2:ThrottleWhileDisarmed,3:Disable while disarmed
```

Comment:
The pattern is CamelCase

Reply:
Might as well fix the DisableRedineGovernor spelling error while you're at it! Redline is missing it's L

Reply:
I did see that the others were. I'd like to know why that is though? I (and many other GCS's) use this information to create a list of drop down/selectbox menus, and it's far more readable to people with white space and not capitalizing everything, and is more amenable to people who don't speak English as a first language.

Reply:
I agree. However, you're 3:1 outNumbered on this example so I suggest you follow the current pattern and make a follow-up PR to change the labeling of all other places that are "wrong".